rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -655,6 +655,42 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     errorLog.readAndClearErrorVector.unsafeRunSync should be(Vector.empty[InterpreterError])
   }
 
+  "Simple let" should "capture and add to the environment" in {
+    implicit val errorLog = new ErrorLog[Task]()
+    val splitRand         = rand.splitByte(0)
+    val result = withTestSpace(errorLog) {
+      case TestFixture(space, reducer) =>
+        val pattern = Send(EVar(FreeVar(0)), List(GInt(7L), EVar(FreeVar(1))), false, BitSet())
+          .withConnectiveUsed(true)
+        val target =
+          Send(EVar(BoundVar(1)), List(GInt(7L), EVar(BoundVar(0))), false, BitSet(0, 1))
+        val matchTerm = Let(
+          Seq(Declaration(pattern, target, freeCount = 2)),
```

Comment:
I would also add a test that shows let working with two declarations. Or maybe change this test and split the two bindings in two separate declarations. 

Also, I don't think the pattern matching part is interesting for this test so I would remove it in order to make the test simpler